docs/man/nsinit.1.md

```diff
@@ -0,0 +1,32 @@
+% nsinit User Manual
+% docker/libcontainer
+% JAN 2015
+
+NAME:
+   nsinit - A new cli application
+
+USAGE:
+   nsinit [global options] command [command options] [arguments...]
+
+VERSION:
+   0.1
+
+COMMANDS:
+   config	display the container configuration
+   exec		execute a new command inside a container
```

Comment:
In the GitHub diff these two commands are more indented than the rest. Is that intentional?